build.gradle

```
@@ -56,16 +56,10 @@ task run(type: JavaExec) {
         System.setProperty "LOG_DIR", "$logDirectory"
         environment "APPLITOOLS_API_KEY", System.getenv("teswiz_APPLITOOLS_API_KEY")
 
-        def configFile = System.getenv("CONFIG")
-        if(null == configFile || !file(configFile).exists()) {
-            println("CONFIG file not provided, or does not exist")
-            assert file(configFile).exists()
-        }
-
-//      You can also specify which config file to use based on the value of RUN_IN_CI as shown below
-//
-//        def configFile = System.getenv("RUN_IN_CI") ? "./configs/calculator_pcloudy_config.properties" : "./configs/jiomeet_local_config.properties"
-//        configFile = System.getenv("CONFIG") ? System.getenv("CONFIG") : configFile
+        def isRunInCI = Boolean.parseBoolean(System.getenv("RUN_IN_CI"))
+        println "isRunningInCI: $isRunInCI"
+        def configFile = System.getenv("RUN_IN_CI") ? "./configs/pcloudy_config.properties" : "./configs/indigoGift_config.properties"
+        configFile = System.getenv("CONFIG") ? System.getenv("CONFIG") : configFile
```

The assert is important to fail fast

---

If the file does not exist at the provided path, the assert will allow the test to fail immediately, which is good. You have removed the assert from the code.